### PR TITLE
OCPBUGS-14900: Use correct SELinux label. Make rename atomic.

### PR DESCRIPTION
**Pull request by pawanpinjarkar**
Signed-off-by: Zane Bitter <zbitter@redhat.com > and Pawan Pinjarkar <ppinjark@redhat.com>

### CODE REVIEW

File: data/data/agent/files/usr/local/bin/issue_status.sh [hunk 1]
```diff
@@ -1,25 +1,19 @@
...
 issue_file() {
-    printf "/etc/issue.d/%s.issue" "$1"
+    printf "/etc/issue.d/%s" "$1"
 }
 
 set_issue() {
     local outfile
     outfile="$(issue_file "$1")"
-    local tmp
-    tmp="$(mktemp)"
```
Comment (zaneb) [inline data/data/agent/files/usr/local/bin/issue_status.sh:11]: One advantage of using mktemp is that there is no way for two processes running this code to interfere with each other. I think we could continue to get the benefits by doing

```
tmp=$(mktemp -p /etc/issue.d/)
```

and then none of the other code needs to change.

**Comment by pawanpinjarkar** [comment]
After fix
Screenshot 2023-07-11 at 10 04 52 AM

**Review by rwsu — COMMENTED** [review]
/lgtm

**Pull request merged**